tests/macros/GroupData.test.js

```diff
@@ -14,7 +14,7 @@ const permittedCharacters = {
     interface: /^[A-Z][\w.]+$/,
     property: /^[\w.]+$/,
     method: /^[\w.()]+$/,
-    event: /^[\w()]+$/,
+    event: /^[\w]+:\ [\w]+$/,
```

Comment:
The space doesn't need to be escaped:
```suggestion
    event: /^[\w]+: [\w]+$/,
```